src/components/Beer.vue

```diff
@@ -19,6 +19,17 @@
         </div>
 
         <p class="beer__description" data-test="beer-description">{{ beer.description }}</p>
+
+        <button
```

Comment:
Same comment as above about toggling text and repeating code

Reply:
Done - moved to computed property 👍